bluebottle/projects/models.py

```diff
@@ -542,6 +542,12 @@ def status_changed(self, old_status, new_status):
 
             bb_track("Project Completed", data)
 
+    def check_task_status(self):
+        if ((not self.is_funding) and
```

Comment:
I tend to add brackets here even tough I know not binds stronger than and. But yeah, they can be removed.